site/layouts/default.hbs

```diff
@@ -31,10 +36,23 @@
 			{{>contentbuttons}}
 			{{>body}}
 {{#if page.deptfeature}}
-	{{>deptfeatures}}
+	{{#is page.pageType "theme-topic"}}{{>deptfeatures-wet-126}}{{/is}}
+	{{#isnt page.pageType "theme-topic"}}{{>deptfeatures}}{{/isnt}}
```

Comment:
I don't understand the difference here between "deptfeatures" and "deptfeatures-wet-126"

Add this include in the page theme-topic template, not from the main layout because it is not reused on anyother page template.

Reply:
I'm confused, deptfeatures was already part of the default.  Would you prefer I add logic to not have it included for this theme-topic page?

Reply:
Let "deptfeatures-wet-126" to be inside the topic-theme page template.

Reply:
This has been implemented.